x-pack/plugins/fleet/server/saved_objects/migrations/to_v7_12_0.ts

```diff
@@ -14,3 +14,14 @@ export const migrateAgentToV7120: SavedObjectMigrationFn<Agent & { shared_id?: s
 
   return agentDoc;
 };
+
+export const migrateAgentPolicyToV7120: SavedObjectMigrationFn<
+  Exclude<AgentPolicy, 'is_managed'>,
+  AgentPolicy
+> = (agentPolicyDoc) => {
+  const isV12 = 'is_managed' in agentPolicyDoc.attributes;
```

Comment:
Do we need the `isV12` here? the migration will only run when it's migrated one time right?

Reply:
Right. It's left over from when I was debug logging; it also felt like a long `if`. I guess it also be `hasManagedAttr` or not a problem to inline it.